src/crimson/net/SocketMessenger.cc

```diff
@@ -116,6 +163,18 @@ void SocketMessenger::set_policy_throttler(entity_type_t peer_type,
   policy_set.set_throttlers(peer_type, throttle, nullptr);
 }
 
+seastar::shard_id SocketMessenger::locate_shard(const entity_addr_t& addr)
+{
+  if (addr.get_family() != AF_INET) {
```

Comment:
if this does not happen in production, probably we can just `assert(addr.get_family() == AF_INET)`.

Reply:
fixed.